platform/linux-generic/odp_ipsec.c

```diff
@@ -2197,6 +2197,21 @@ int odp_ipsec_stats(odp_ipsec_sa_t sa, odp_ipsec_stats_t *stats)
 	return 0;
 }
 
+int odp_ipsec_stats_multi(odp_ipsec_sa_t sa[], odp_ipsec_stats_t stats[], int num)
+{
+	int ret, i;
+
+	ODP_ASSERT(NULL != stats);
+
+	for (i = 0; i < num; i++) {
+		ret = odp_ipsec_stats(sa[i], &stats[i]);
+		if (ret)
+			return ret;
+	}
+
+	return num;
```

Comment:
I failed to notice this in the previous version. According to the API spec, the return value should be 0 on success.

Reply:
updated it